docs/L3AF_technical_charter.md

```diff
@@ -0,0 +1,128 @@
+# Technical Charter (the “Charter”) for L3AF Project a Series of LF Projects, LLC
+
+_Adopted February 16, 2022_
+
+This Charter sets forth the responsibilities and procedures for technical contribution to, and oversight of, the
+L3AF open source project, which has been established as L3AF Project a Series of LF Projects, LLC (the “Project”).
+LF Projects, LLC (“LF Projects”) is a Delaware series limited liability company. All contributors (including
+committers, maintainers, and other technical positions) and other participants in the Project (collectively,
+“Collaborators”) must comply with the terms of this Charter.
+
+## 1. Mission and Scope of the Project
+
+<ol type="a">
+   <li>The mission of the project is to empower the Linux kernel networking eBPF community by providing a cloud and vendor agnostic platform to orchestrate eBPF programs into the kernel, thereby enabling an eco-system that delivers Kernel Function As A Service.</li>
```

Comment:
I agree this is what the charter says so approving, but I think the mission should be expanded beyond empowering just the Linux kernel networking eBPF community, to empowering the eBPF community as a whole (not just networking, not just Linux, not just kernel).